makePoplogCommander.sh

```diff
@@ -476,46 +664,76 @@ void extendPath( char * prefix, char * path, char * suffix ) {
     free( buff );
 }
 
-void setUpEnvironment( char * base, int flags, Chain envv ) {
+****
+
+# Here we create three functions for setting the environment variables that
+# are unique to the build variants: nox, xm, xt. These will be called
+# nox_setUpEnvVars, xm_setUpEnvVars, xt_setUpEnvVars.
+VARIANTS='nox xt xm'
```

Comment:
I'm not overly keen on the use of word splitting to iterate over what is effectively an array.
I'd suggest replacing this with `VARIANTS=( nox xt xm )` and replacing the for loops with `for variant in "${VARIANTS[@]}"

Reply:
Yes, I thought you would pick me up on this one ... I just didn't know the way to do it. Ugh bash magic. Fixing.